jobs/check_solvers.ts

```diff
@@ -0,0 +1,37 @@
+import { Community, User } from '../models';
+import { Telegraf } from 'telegraf';
+import { MainContext } from '../bot/start';
+import { logger } from '../logger';
+
+const MESSAGES: number = parseInt(process.env.COMMUNITY_MESSAGES || '5');
+
+exports.checkSolvers = async (ctx: MainContext, bot: Telegraf<MainContext>): Promise<void> => {
+    try {
+        const communities = await Community.find({ isDisabled: false });
+
+        for (const community of communities) {
+            const solvers = await User.find({ default_community_id: community._id, role: 'solver' });
+
+            if (solvers.length === 0) {
+                community.messagesSent += 1;
+
+                if (community.messagesSent >= MESSAGES) {
+                    community.isDisabled = true;
+                    await community.save();
+                    logger.info(`Community ${community._id} has been disabled due to lack of solvers.`);
+                } else {
+                    await community.save();
+                    const admin = await User.findOne({ tg_id: community.creator_id, admin: true });
```

Comment:
`user.admin` is only used for global admin, not community admins, we don't need to add it on this `findOne()` request